CHANGELOG.latest.md

```diff
@@ -1,2 +1,79 @@
-### Dependency Updates
-* [AUTOMATIC] iOS 4.17.7 => 4.17.8 (#361) via RevenueCat Git Bot (@RCGitBot)
+The first beta of **RevenueCat Purchases Hybrid Common v5** is here!! 😻
+
+This latest release updates the Android SDK dependency from v5 to [v6](https://github.com/RevenueCat/purchases-android/releases/tag/6.0.0) to use BillingClient 5. This version of BillingClient brings an entire new subscription model which has resulted in large changes across the entire SDK.
+
+### Migration Guides
+- See [Android Native - 5.x to 6.x Migration](https://www.revenuecat.com/docs/android-native-5x-to-6x-migration) for a
+  more thorough explanation of the new Google subscription model announced with BillingClient 5 and how to take
+  advantage of it in V6. This guide includes tips on product setup with the new model.
+
+### New `SubscriptionOption` concept
+
+#### Purchasing
+
+Now, in Hybrid Common v5, an Google Play Android `Package` or `StoreProduct` represents a duration of a subscription and contains all the ways to
+purchase that duration -- any offers and its base plan. Each of these purchase options are `SubscriptionOption`s.
+When passing a `Package` to `purchasePackage()` or `StoreProduct` to `purchaseStoreProduct()`, the SDK will use the following logic to choose which
+`SubscriptionOption` to purchase:
+*   - Filters out offers with "rc-ignore-offer" tag
+*   - Uses `SubscriptionOption` with the longest free trial or cheapest first phase
+*   - Falls back to base plan
```

Comment:
one thing i was thinking about (for native too) is that maybe we should explain we'll never apply an offer the user isn't eligible for...

our customers shouldn't have to know what google passes to us, and i don't think it's immediately clear